pkg/web/templates/partials/secure_envelope/secure_envelope_list.html

```diff
@@ -67,8 +67,8 @@ <h3>On-chain Transaction Info</h3>
     {{ if .IsError }}
     <div class="collapse-title bg-red-200 text-xl font-medium">
       {{ else }}
-      <div class="collapse-title bg-green-200 text-xl font-medium">
```

Comment:
This was a style that I was testing. I think only highlighting error envelopes will be preferable as we expect to have less of those.